symlink_bootstrap.py

```diff
@@ -0,0 +1,41 @@
+# RUN THIS ONCE TO SYMLINK _bootstrap.py into runnable script folders
+import sys
+from pathlib import Path
+
+# list of sub directories of the project containing scripts that are run directly
+# NOTE: do NOT need to add folders like `utils/` or `config/` since they are never directly called
+WORKING_DIRS = [
```

Comment:
It might be helpful to provide a note in the `README` or a new `CONTRIBUTING.md` file explaining to modify this as needed. Since people may not think to look at this file.